README.md

```diff
@@ -238,6 +248,8 @@ CREATE EXTENSION set_user;
   * set_user.block_log_statement = on
 * Block switching to a superuser role
   * set_user.block_superuser = on
+* Whitelist users for superuser escalation
```

Comment:
How do block_superuser = on and superuser_whitelist = 'non-empty' interact? If block_superuser = on does that mean effectively superuser_whitelist is ignored? If block_superuser = off and superuser_whitelist = '' then anyone able to run set_user_u can escalate but if superuser_whitelist = 'non-empty' then only whitelist users can escalate with set_user_u? 

I'm sure I can figure this out by inspecting the code, but we need to be clear in our minds exactly what behavior we want and in the docs what behavior is implemented.